cfgov/unprocessed/js/modules/FlyoutMenu.js

```diff
@@ -65,7 +71,7 @@ function FlyoutMenu( element, triggerSel, contentSel, altTriggerSel ) {
    */
   function expand() {
     if ( !_isExpanded && !_isAnimating ) {
-      this.dispatchEvent( 'toggle', { target: this } );
+      _deferFunct = standardType.noopFunct;
```

Comment:
Why do you need to set the `deferFunct` to ` standardType.noopFunct;` everytime `expand` is called?